src/nni_manager/yarn.lock

```diff
@@ -2716,15 +2717,17 @@ tslib@^1.8.0, tslib@^1.8.1:
   version "1.9.3"
   resolved "https://registry.yarnpkg.com/tslib/-/tslib-1.9.3.tgz#d7e4dd79245d85428c4d7e4822a79917954ca286"
 
-tslint-microsoft-contrib@^5.1.0:
-  version "5.1.0"
-  resolved "https://registry.yarnpkg.com/tslint-microsoft-contrib/-/tslint-microsoft-contrib-5.1.0.tgz#777c32d51aba16f4565e47aac749a1631176cd9f"
+tslint-microsoft-contrib@^6.0.0:
+  version "6.1.0"
+  resolved "http://registry.npm.taobao.org/tslint-microsoft-contrib/download/tslint-microsoft-contrib-6.1.0.tgz#cc487575f2cdb8fe3774bf3f2ba61ff61dc7856e"
```

Comment:
can you explain why you change the source here?

Reply:
Sorry, I don't know why files like 'yarn.lock' have been changed.

I just modified two files:
- src/webui/src/components/overview/SuccessTable.tsx
- src/webui/package.json

and other files like *package-lock.json*、*setup.pye*、*yarn.lock* might be auto generated.You may add them to *.gitignore* to avoid these unexcepted changes.

Reply:
@lvybriage

Reply:
@xuehui1991 It's not webui's code. These files are in nni_manager.

Reply:
sorry. @goooxu

Reply:
@ymlgithub Could you revert the unrelated changes?